app/src/main/res/drawable/ic_placeholder_no_entries.xml

```diff
@@ -0,0 +1,76 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Nit, and for lack of a better place to put this comment: Is `_placeholder_` the right name for this file (and it's `png` alternatives)?  In US English, it most often denotes "something to use until the real thing is available".  I don't have a better suggestion, other than dropping `placeholder` if possible.

Reply:
I agree, I'll change it to "ic_no_entries"